### ISSUE TITLE
Problemas cargando el modelo VENSIM

### ISSUE THREAD

**Issue by SabineEgerer** [julienmalard/Tinamit#25]
Dear Julian,

I am working in the same project as Andrea Fajardo at the Climate Service Center Germany (GERICS) who recently contacted you.
I succesfully could run Tinamit on my Linux machine but when loading the Vensim model (I tried with the Ejemplo_basico: modelo.estab_mds('Prueba dll.vpm') using the IPA) I got an error:

  File "/home/egerer/Programme/anaconda3/lib/python3.6/site-packages/tinamit/EnvolturaMDS/__init__.py", line 50, in generar_mds
    '{}').format(''.join(['\n\t{}: {}'.format(env, e) for env, e in errores.items()])))

IndexError: tuple index out of range

Do you have an idea what may be the cause for the error?

Thanks and all the best,
Sabine

**Información adicional**
 - Versión de Python [p. ej., 3.6.1]
 - Versión de Tinamït [p. ej., 2.0.1, ``master``, etc.]

**Comment by julienmalard** [comment]
Favor incluir la información adicional: versión de Python y de Tinamït.
La versión se puede encontrar con 
```python
import tinamit
tinamit.__versión__
```
---
Bitte einschließen sie die zusätzliche Informationen: Pythonversion und Tinamitversion.
Das können Sie finden mit:
```python
import tinamit
tinamit.__versión__
```

**Comment by SabineEgerer** [comment]
I used Python Version 3.6 and Tinamit Version 1.4.0

**Comment by julienmalard** [comment]
¿Sería que ya se arregló? Si es el caso favor confirma y marcaré este reporto como resuelto.

**Comment by SabineEgerer** [comment]
The problem is solved. I followed the instructions as described here:
https://github.com/julienmalard/Tinamit/blob/master/README.rst
but when loading the SD model I had to go in the directory where the Vensim model is located and I had to use the .mdl ending instead:
modelo.estab_mds('Prueba dll.mdl').
This might be because I am working with Linux environment and not Windows.
However, I got problems when loading the BF model (see next Issue).
